pkg/secretstorage/awsstorage/aws.go

```diff
@@ -97,35 +102,39 @@ func (s *AwsSecretStorage) Get(ctx context.Context, id secretstorage.SecretID) (
 			if notFoundErr, ok := awsError.(*types.ResourceNotFoundException); ok {
 				secretData, migrationErr := s.tryMigrateSecret(ctx, id) // this migration is just temporary
 				if migrationErr != nil {
-					dbgLog.Error(migrationErr, "something went wrong during migration")
+					msgLog.Error(migrationErr, "something went wrong during migration")
 				}
 				if secretData != nil {
-					dbgLog.Info("secret successfully migrated", "secretid", id)
+					msgLog.V(logs.DebugLevel).Info("secret successfully migrated", "secretid", id)
 					return secretData, nil
 				} else {
-					dbgLog.Info("secret not found in aws storage", "err", notFoundErr.ErrorMessage())
-					return nil, fmt.Errorf("%w: %s", secretstorage.NotFoundError, notFoundErr.Error())
+					msgLog.Info("secret not found in aws storage", "err", notFoundErr.Error())
```

Comment:
`... "error", notFoundErr.Error()` to be consistent with the rest of the log messages below?

Reply:
ok